pkg/operator/deploy/deploy.go

```diff
@@ -148,18 +160,18 @@ func (o *operator) resources() ([]runtime.Object, error) {
 			},
 			Spec: arov1alpha1.ClusterSpec{
 				ResourceID: o.oc.ID,
-				ACRName:    o.env.ACRName(),
+				ACRDomain:  o.env.ACRDomain(),
```

Comment:
@jim-minter Newly deployed ARO operators will set ACRDomain.

Reply:
Yup, and it's set on upgrade too.